tests/generative/data_setup.py

```diff
@@ -48,10 +47,33 @@ def build_table(name, schema, patient_id_column, ids, registry):
     return class_
 
 
-def make_backend(patient_id_column_):
-    class Backend(BaseBackend):
-        backend_id = "gen-test-backend"
-        query_engine_class = "not-needed"
-        patient_join_column = patient_id_column_
+def _make_backend(table_names, schema, patient_id_column):
+    tables = _backend_tables(schema, table_names)
+    class_vars = {
+        "backend_id": "gen-test-backend",
+        "query_engine_class": "not-needed",
+        "patient_join_column": patient_id_column,
+    } | tables
+    return type("Backend", (backends.BaseBackend,), class_vars)
```

Comment:
:trophy:

Reply:
My golden rule of Python development is that you're not _really_ having a good time until you're using the three argument form of `type()`.